quipucords/scanner/network/runner/roles/date/molecule/default/molecule.yml

```diff
@@ -11,3 +11,7 @@ provisioner:
   name: ansible
 verifier:
   name: ansible
+#lint: |
+#  set -e
+#  yamllint .
+#  ansible-lint .
```

Comment:
:thinking: why is this commented? :thinking:

Reply:
Running ansible lint on this project raised a crazy amount of errors. Some really easy to fix, other's a bit more complex like: "replace ignore_errors:yes for failed_when:{{reason for failing}}".Do we have to map out errors for every task? How many different errors are there? Should we ignore this specific rule? All unanswered questions that I plan to bring up to the team when are a bit more advanced on the process of implementing this.

Reply:
* Enable yamllint first and see what it catches, see if it worths running now.
* * Is there any configuration file for ansible-lint so we can make a config to avoid the linter being rather grumpy at this time?

Reply:
I ran yamllint and it pointed 0 errors. Yes, we can config what we want to enable/disable on ansible-lint.